_posts/2018-11-02-JOIN-2018.md

```diff
@@ -0,0 +1,507 @@
+---
+layout: post
+authors: [tom_van_den_bulck,tim_de_grande]
+title: 'JOIN 2018'
+image: /img/join-2018/join-2018.jpg
+tags: [Join,Java,Agile,Conference]
+category: Conference
+comments: true
+---
+
+# JOIN is back!
+
+As has become tradition, Ordina JWorks organised the 6th annual JOIN conference on Oct 4th.
+This edition was the 6th and once again, it was bigger and better than the previous editions.
```

Comment:
Two times 6th edition right after one another. Suggestion: This edition was once again bigger and better ...